zenoh/src/net/runtime/adminspace.rs

```diff
@@ -622,15 +622,8 @@ fn local_data(context: &AdminContext, query: Query) {
     }
 
     tracing::trace!("AdminSpace router_data: {:?}", json);
-    let payload = match ZBytes::try_from(json) {
-        Ok(p) => p,
-        Err(e) => {
-            tracing::error!("Error serializing AdminSpace reply: {:?}", e);
-            return;
-        }
-    };
     if let Err(e) = query
-        .reply(reply_key, payload)
+        .reply(reply_key, json.to_string())
```

Comment:
`to_string()` should not be used for encoding a json. The previous code performs the correct `try` for serialization.